### PR TITLE
Centralize and localize messages

### PR DESCRIPTION
**Pull request by allawala**
https://github.com/allawala/service-chassis/issues/46

### CODE REVIEW

File: src/main/scala/allawala/chassis/core/module/ChassisModule.scala [hunk 1]
```diff
@@ -5,32 +5,49 @@ import javax.inject.Named
...
 
 import scala.concurrent.ExecutionContext
 
-
-class ChassisModule extends AbstractModule with ScalaModule {
+abstract class ChassisModule extends AbstractModule with ScalaModule {
   override def configure(): Unit = {
     install(new ConfigModule)
     install(new HttpModule)
+    bindHealthModule()
```
Comment (hoffbeam) [inline src/main/scala/allawala/chassis/core/module/ChassisModule.scala]: why did you move these to a seperate method that is doing exactly this?

Comment (allawala) [reply src/main/scala/allawala/chassis/core/module/ChassisModule.scala]: So that the implementing services can overwrite as needed
